src/elements/content-sidebar/SidebarNav.scss

```diff
@@ -63,3 +63,88 @@
         }
     }
 }
+
+.is-responsive-web {
+    @include breakpoint($small-screen) {
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 46px;
+            margin-right: 0;
+        }
+    }
+
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNav,
+        .bcs-SidebarNav-tabs {
+            flex-direction: row;
+        }
+
+        .bcs-SidebarNav-main {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .bcs-NavButton::before {
+            right: 0;
+            bottom: 0;
+            left: 0;
+            height: 2px;
+        }
+
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 48px;
+            height: 48px;
+            margin-right: 12px;
+        }
+
+        .bdl-AdditionalTabs > .bdl-AdditionalTab:not(.bdl-is-overflow),
+        .bdl-AdditionalTabs > .bdl-AdditionalTabPlaceholder {
+            display: none;
+        }
+
+        .bdl-AdditionalTabs > :last-child,
```

Comment:
What are we selecting here?

Reply:
The last child in the `bdl-AdditionalTabs` is the ... button for More Apps. While the last child in `bcs-SidebarNav-secondary` is the Sign button. Setting the `margin-right: 0` for last child of both of these avoids pushing out the footer button:

#### Without setting `margin: 0`
![Screen Shot 2022-07-13 at 5 13 17 PM](https://user-images.githubusercontent.com/6400298/178858121-d8af155a-1c58-4feb-a049-1dadf8046798.png)